[XWI-60] Documentation of admin guide (#23884)
opf/openproject: docs/system-admin-guide/wikis/README.md

---
sidebar_navigation:
  title: Wikis
  priority: 835

> description: File storages in OpenProject.

Reviewer: ```suggestion
description: Wiki providers in OpenProject.
```

keywords: wikis, xwiki, wiki integration, wiki provider, internal wiki
---

# Wikis

Under *Administration → Wikis* you can manage the application wide available wikis. Generally, there is a distinction
between internal and external wikis. The application provides the possibility to configure wiki providers, which are

> considered to have multiple wikis inside. Wikis are globally available in the application.

Reviewer: ```suggestion
considered to contain multiple wikis. Wikis are globally available in the application.
```

## Internal wiki

Under *Administration → Wikis → Internal wiki* you can enable or disable the internal wiki provider. If the wiki
provider is enabled, each project can have its own wiki. Access level is defined by the member's role in the project.

> *Insert screenshot of internal wiki configuration*

> 

Reviewer: @Kharonus can we clarify here that this is does not mean enabling/disabling the Wiki module under project settings, but activating it for being able to use it in the CKEditors within the project? At least this is how i understood it.

Author: I added more explanatory text, I hope it reads more clear now?
Disabling the internal provider fully prevents any usage of internal wikis in projects. It does NOT prevent linking wiki pages in work packages or inside the CKEditor, at least not to external wikis. ;) The internal ones are hidden then, of course.

## External wikis

Under [Wiki providers](./wiki-providers) you can configure external wiki providers. External providers outsource the

> user permission to the external server, thus it is required to connect the OpenProject user to a user on the external

Reviewer 2: I'd keep our language consistent here and not talk about a "server", but keep it with "provider".

```suggestion
user permission to the external provider, thus it is required to connect the OpenProject user to a user on the external
```

Author: Chose a bit different words, but I hope I improved it.
